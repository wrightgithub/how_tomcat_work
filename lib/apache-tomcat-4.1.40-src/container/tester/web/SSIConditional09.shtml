1
<!--#if expr="1=2" -->
a
##<!--#if expr="1=2" -->
b
##<!--#else -->
c
##<!--#endif -->
d
<!--#elif expr="2=2" -->
e
##<!--#if expr="2=2" -->
f
####<!--#if expr="1=2" -->
**11
####<!--#elif expr="2=3" -->
**22
####<!--#endif -->
**33
##<!--#else -->
g
##<!--#endif -->
h
<!--#else -->
i
##<!--#if expr="1=1" -->
j
##<!--#else -->
k
##<!--#endif -->
l
<!--#endif -->
#
now we test extra #endif commands
<!--#endif -->
n
<!--#endif -->
o