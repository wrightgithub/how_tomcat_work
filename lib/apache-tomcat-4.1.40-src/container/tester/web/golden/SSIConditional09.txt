1

e
##
f
####
**33
##
h

#
now we test extra #endif commands

n

o